src/modules/ProductLicense.rb

```diff
@@ -377,14 +379,21 @@ def GetLicenseDialog(languages, license_language, licenses, id, spare_space)
               ReplacePoint(
                 Id(:printing_hint),
                 Label(
-                  # TRANSLATORS: addition license information
-                  # %1 is replaced with the filename
-                  Builtins.sformat(
-                    _(
-                      "If you want to print this EULA, you can find it\non the first media in the file %1"
-                    ),
-                    @license_file_print
-                  )
+                  @license_on_installed_system ?
+                    # TRANSLATORS: addition license information
+                    # %s is replaced with the directory name
+                    format(
```

Comment:
Instead of `format(_("txt"), variable)` you can use the `%` operator: `_("txt") % variable`